src/style/ImpactCalculator.less

```diff
@@ -107,22 +91,19 @@
 				border: none;
 				border-radius: 0;
 				box-shadow: none;
+				//font-size and weight set here to match figma - bold & 28px
```

Comment:
OK - but is that one of the standard "top-level" sizes that gets used in a few places, or is this specific to this widget only? (and if so, why?)

Reply:
It looks like that size is specific to this widget only and it is only for the green values in the impact calculator. Looks like Brian designed it that way for emphasis?